snapcraft.yaml

```diff
@@ -16,10 +16,11 @@ parts:
     source: https://github.com/juju/juju.git
     source-type: git
     build-packages: [gcc]
-    #If you want to grab a specific revision tag, include it here
```

Comment:
Yes, you could make the source local instead. Including an example of this would be useful.